bugbug/commit_features.py

```diff
@@ -202,6 +217,10 @@ def __init__(self, feature_extractors, cleanup_functions):
         self.cleanup_functions = cleanup_functions
 
     def fit(self, x, y=None):
+        for item in self.feature_extractors:
+            if item.__class__ == files:
```

Comment:
Instead of making it specific for `files`, check if the item has a `fit` function